README.md

```diff
@@ -262,27 +294,27 @@ type FeeQuote struct {
 
 ```go
 type SubmitTxResponse struct {
- BlockHash string `json:"blockHash,omitempty"`
- BlockHeight int64 `json:"blockHeight,omitempty"`
- ExtraInfo string `json:"extraInfo,omitempty"`
- Status int `json:"status,omitempty"`
- Title string `json:"title,omitempty"`
- TxStatus TxStatus `json:"txStatus,omitempty"`
+    BlockHash string `json:"blockHash,omitempty"`
+    BlockHeight int64 `json:"blockHeight,omitempty"`
+    ExtraInfo string `json:"extraInfo,omitempty"`
+    Status int `json:"status,omitempty"`
+    Title string `json:"title,omitempty"`
+    TxStatus TxStatus `json:"txStatus,omitempty"`
 }
 ```
 
 #### Transaction
 
   ```go
 type Transaction struct {
-  CallBackEncryption string `json:"callBackEncryption,omitempty"`
-  CallBackToken string `json:"callBackToken,omitempty"`
-  CallBackURL string `json:"callBackUrl,omitempty"`
-  DsCheck bool `json:"dsCheck,omitempty"`
-  MerkleFormat string `json:"merkleFormat,omitempty"`
-  MerkleProof bool `json:"merkleProof,omitempty"`
-  RawTx string `json:"rawtx"`
-  WaitForStatus TxStatus `json:"waitForStatus,omitempty"`
+    CallBackEncryption string `json:"callBackEncryption,omitempty"`
+    CallBackToken string `json:"callBackToken,omitempty"`
+    CallBackURL string `json:"callBackUrl,omitempty"`
+    DsCheck bool `json:"dsCheck,omitempty"`
+    MerkleFormat string `json:"merkleFormat,omitempty"`
+    MerkleProof bool `json:"merkleProof,omitempty"`
+    Hex string `json:"rawtx"`
```

Comment:
Not sure if it's still a rawtx or hex in json annotation.

Reply:
It's hex now, thanks for spotting that!